tests/test_project.py

```diff
@@ -304,10 +337,13 @@ def test_find_jobs_JobsCursor_contains(self):
                 assert self.project.open_job(sp) in cursor_first
             else:
                 assert self.project.open_job(sp) not in cursor_first
-        cursor_doc = self.project.find_jobs(doc_filter={"test": True})
+        cursor_doc = self.project.find_jobs({"doc.test": True})
```

Comment:
Please duplicate the test instead of altering it to ensure that we do not introduce a regression in future releases.